ychat-core/build.gradle.kts

```diff
@@ -0,0 +1,87 @@
+plugins {
```

Comment:
What about adding to this module the Explicit Mode for library authors?

https://kotlinlang.org/docs/whatsnew14.html#explicit-api-mode-for-library-authors

Reply:
explicitApi() has been added to the `:core` module. As for the `:ychat` module, I will be adding it to the branch where I'm performing the entire modularization refactor.